activitysim/abm/models/school_escorting.py

```diff
@@ -533,9 +554,26 @@ def school_escorting(
 
         if estimator:
             estimator.write_model_settings(model_settings, model_settings_file_name)
-            estimator.write_spec(model_settings)
-            estimator.write_coefficients(coefficients_df, model_settings)
+            estimator.write_spec(model_settings, tag=stage.upper() + "_SPEC")
+            estimator.write_coefficients(
+                coefficients_df, file_name=stage.upper() + "_COEFFICIENTS"
+            )
             estimator.write_choosers(choosers)
+            estimator.write_alternatives(alts, bundle_directory=True)
+
+            # FIXME #interaction_simulate_estimation_requires_chooser_id_in_df_column
+            #  shuold we do it here or have interaction_simulate do it?
+            # chooser index must be duplicated in column or it will be omitted from interaction_dataset
+            # estimation requires that chooser_id is either in index or a column of interaction_dataset
+            # so it can be reformatted (melted) and indexed by chooser_id and alt_id
+            assert choosers.index.name == "household_id"
+            assert "household_id" not in choosers.columns
+            choosers["household_id"] = choosers.index
+
+            # FIXME set_alt_id - do we need this for interaction_simulate estimation bundle tables?
+            estimator.set_alt_id("alt_id")
```

Comment:
The question in the FIXME - does it remain as a question?

Reply:
Yes, this was copied directly from the non-mandatory tour frequency model since I am keeping this consistent with what was done for that model (see [here](https://github.com/ActivitySim/activitysim/blob/main/activitysim/abm/models/non_mandatory_tour_frequency.py#L297).  My guess is that it has been around since estimation mode was first developed. This note here is to help us keep track of this issue across all locations in the code. (It shows up in the vehicle type choice model as well.)